code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -105,6 +105,17 @@
 		but avoid actively interfering with the station, you are required to adminhelp and request permission to board the main station.</span>")
 	else
 		new_spawn.mind.store_memory("<b>Serve [owner.real_name], your creator.</b>")
+		if(owner.mind.special_role)
```

Comment:
are there any issues with converting this to an early return?

Reply:
Nope!

Reply:
Wait no I lied, that just makes some errors with non-antags trying to make golems, sorry